connectors/rdbms/r2dbc/build.gradle.kts

```diff
@@ -1,8 +1,10 @@
-
+import Dependencies.RiverCore
 object Version {
     const val Kotlin = "1.7.20"
     const val Coroutine = "1.6.4"
     const val Slf4j = "1.7.36"
+    const val R2dbcSpi = "1.0.0.RELEASE"
+    const val H2 = "1.0.0.RELEASE"
```

Comment:
In this case, I think both `R2dbcSpi` and `H2` are from the same group and follow the same versions. I think you could use only `const val R2dbc = "1.0.0.RELEASE"` and use it for both dependencies.

Reply:
Also, you should move these dependencies to `buildSrc/src/main/kotlin/Dependencies.kt` and remove this `Version` object, my bad, I should have this removed. 😅